src/util/trackingUtil.js

```diff
@@ -81,7 +81,10 @@ export const getAllDimensions = (
     4: contentTypeLabel,
     5: subject ? subject.name : undefined,
     6: topicPath && topicPath[0] ? topicPath[0].name : undefined,
```

Comment:
For alle disse kunne vi vel innført chaining? 
```suggestion
    6: topicPath?.[0]?.name || undefined,
```